script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -1,62 +1,62 @@
 <Page1>
 ...I watch the crowd in front of the station in a daze.
-It's a little past two in the afternoon.
+It's a little past two.
 The bustle of the city continues unabated.
 A large number of people pass by.
-I don't know their names or their identities, but each of them is heading towards their own destinations without looking at anyone walking right next to them.
+I don't know their names or their identities, but each of them is bound to their own destination without giving a thought to any of the people walking right beside them.//名前も素性も知らない彼らは、すぐとなりを歩いている誰かに視線を向ける事なく、自分だけの目的地に向かって行く。quite liberally translated here but I think it maintains the same sense.
 <Page2>
 Despite all these people being in the same place, they all continue on within the single-minded limits of their own personal journey.
 Like how everyone is their own main character,
-they are living each day that passes with themselves at center stage.
+Everyone lives each day with themselves at center stage.//　彼らは彼らが主役の一日を過ごしている。Dropping "過ごしている" because I think it reads smoother without it.
```

Comment:
Previous line ends in comma, either no caps here or end sentence above.

Reply:
line above should be its own sentence given those are disjointed thoughts in english.